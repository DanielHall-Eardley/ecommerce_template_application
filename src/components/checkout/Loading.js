import React from 'react'
import styles from './Loading.module.css'
import sprite from '../../sprite.svg'

export default props => {
  return (
    <div className={styles.overlay}>
      <svg className={styles.spinner}>
        <use xlinkHref={sprite + 'sprite.svg#icon-loop2'}></use>
      </svg>
      <p className={styles.msg}>{props.msg}</p>
    </div>
  )
}