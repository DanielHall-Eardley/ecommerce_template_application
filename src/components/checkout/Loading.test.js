import React from 'react'

import {shallow} from 'enzyme'
import Loading from './Loading'

it('test CheckoutSummary snapshot', () => {
  const wrapper = shallow(<Loading msg='A message'/>)
  expect(wrapper).toMatchSnapshot()
})