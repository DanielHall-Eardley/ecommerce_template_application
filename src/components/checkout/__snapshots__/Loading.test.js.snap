// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test CheckoutSummary snapshot 1`] = `ShallowWrapper {}`;
